label_studio/data_import/api.py

```diff
@@ -459,7 +459,8 @@ def get(self, *args, **kwargs):
         file = file_upload.file
         if file.storage.exists(file.name):
             content_type, encoding = mimetypes.guess_type(str(file.name))
-            content_type = content_type or 'application/octet-stream'
+            if content_type not in ['image/jpeg', 'image/gif', 'image/png', 'application/pdf']:
```

Comment:
Looks like it may break a playback for audio, video and other media formats that browser supports natively - please check the list of formats here: https://developers.google.com/cast/docs/media

Reply:
Does label studio allow labeling audio and video content? 

I thought this endpoint `UploadedFileResponse` was only a convenience for viewing uploaded content? 

Does it have a function for the application? I don't think we need to allow this end point at all...

Reply:
> Does label studio allow labeling audio and video content? 

Sure, check all the templates on the Create Project dialog. 

> I thought this endpoint UploadedFileResponse was only a convenience for viewing uploaded content?

This endpoint provides uploaded data not only for viewing, but for annotators during labeling too. It's important part especially for our opensource community users. Enterprise clients use s3/gcs/azure storages more, but sometimes they like to switch to uploaded files. 

btw - after this fix users won't be able to use SVG files in their labeling workflows. It seems to fix this issue we need to parse uploaded SVG file and remove its dangerous parts...